### PR TITLE
Core sets fix

### PR DESCRIPTION
**Pull request by johncoxon**
This contains a fix for #26 which adds the two scenario packs to the collection builder.

It also:
* Fixes the name of Hotshots and Aces II to the one used by AMG
* Fixes an issue where the version of "Pops" Krail in Hotshots and Aces II was incorrect
* Adds the Battle of Yavin subtitle to the relevant Imperial pilots
* Updates the Hotshots and Aces II wave number to match the one on the X-Wing wiki

### CODE REVIEW

**Comment by johncoxon** [comment]
Does this need any edits before it can be merged? I can make tweaks if I've made any mistakes 😊

**Comment by johncoxon** [comment]
@andrelind Can this be merged? Do I need to do anything?

**Review by andrelind — APPROVED** [review]

**Pull request merged**
